miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/LibraryRestController.java

```diff
@@ -121,18 +121,10 @@ public LibraryDto getLibraryById(@PathVariable Long libraryId) throws IOExceptio
     if (l == null) {
       throw new RestException("No library found with ID: " + libraryId, Status.NOT_FOUND);
     }
-    LibraryDto dto = Dtos.asDto(l);
+    LibraryDto dto = Dtos.asDto(l, false);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "dto". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)